R/check.r

```diff
@@ -109,16 +113,18 @@ check_r_cmd <- function(built_path = NULL, cran = TRUE, check_version = FALSE,
 
   opts <- paste(paste(opts, collapse = " "), paste(args, collapse = " "))
 
-  env_vars <- NULL
   # Setting these environment variables requires some care because they can be
   # be TRUE, FALSE, or not set. (And some variables take numeric values.) When
   # not set, R CMD check will use the defaults as described in R Internals.
-  env_vars <- c(
+  env_vars <- c(env_vars,
     if (cran) cran_env_vars(),
     if (check_version) c("_R_CHECK_CRAN_INCOMING_" = "TRUE", aspell_env_var()),
     if (!force_suggests) c("_R_CHECK_FORCE_SUGGESTS_" = "FALSE")
   )
 
+  # remove duplicated vars, keeping only the first instance
+  env_vars <- env_vars[!duplicated(names(env_vars))]
```

Comment:
This seems backwards to me - normally you'd keep the last instance. Can you please flip around with `fromLast`?